usage/icsp/icsp.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-goinfra/pkg/icsp"
+)
+
+func main() {
+	apiClient := clients.New("")
+
+	// Define new ICSPBuilder
+	icspbuilder := icsp.NewICSPBuilder(apiClient, "testicsp", "mysource", []string{"mymirror.io"})
+
+	// Create new ImageContentSourcePolicy from icspbuilder.
+	_, err := icspbuilder.Create()
+	if err != nil {
+		fmt.Println(err.Error())
+	}
+	// Add new source to the ImageContentSourcePolicy definition
+	icspbuilder.WithRepositoryDigestMirror("newsource", []string{"Sasha.io", "Trey.io", "Chad.io"})
```

Comment:
Typically we chain methods which would be useful to display in the example. I think you can combine this with update:
```
_, err = icspbuilder.WithRepositoryDigestMirror("newsource", []string{"Sasha.io", "Trey.io", "Chad.io"}).Update()
```
Also, since this is a public repo can you use different example mirror names 😂

Reply:
Done